File: src/test/dev/alphaserpentis/examples/coffeecore/java/custom/handler/CustomServerData.java
package dev.alphaserpentis.examples.coffeecore.java.custom.handler;

import dev.alphaserpentis.coffeecore.data.server.ServerData;

public class CustomServerData extends ServerData {
    private String customData = "This is custom data!";

    public CustomServerData() {}

    public CustomServerData(boolean onlyEphemeral) {
        super(onlyEphemeral);
    }

    public CustomServerData(boolean onlyEphemeral, String customData) {
        super(onlyEphemeral);
        this.customData = customData;
    }

    public void setCustomData(String customData) {
        this.customData = customData;
    }

    public String getCustomData() {
        return customData;
    }
}

File: src/test/kotlin/dev/alphaserpentis/examples/coffeecore/custom/handler/CustomServerData.kt
package dev.alphaserpentis.examples.coffeecore.kotlin.dev.alphaserpentis.examples.coffeecore.custom.handler

import dev.alphaserpentis.coffeecore.data.entity.ServerData

class CustomServerData : ServerData {
    var customData = "This is custom data!"

    constructor()
    constructor(onlyEphemeral: Boolean) : super(onlyEphemeral)
    constructor(onlyEphemeral: Boolean, customData: String) : super(onlyEphemeral) {
        this.customData = customData
    }
}
